environment-dev.yml

```diff
@@ -2,54 +2,43 @@ name: pyjanitor-dev
 channels:
   - conda-forge
 dependencies:
-  - python=3.9
+  - python>=3.6
+  # required
+  - pandas
+  - pandas-flavor
+  - multipledispatch
+  - scipy
```

Comment:
`scipy` I thought also could as an optional dependency.
there are only two scripts that use scipy
- `math.py`
- `functions/impute.py`